CONTRIBUTORS.md

```diff
@@ -3,12 +3,12 @@
 | Name                       | GitHub                                                           | Social                                                        |
 | :------------------------- | :--------------------------------------------------------------- | :------------------------------------------------------------ |
 | **Lars-Magnus Skog**       | [**@ralphtheninja**](https://github.com/ralphtheninja)           | [**@ralph@social.weho.st**](https://social.weho.st/@ralph)    |
-| **Rod Vagg**               | [**@rvagg**](https://github.com/rvagg)                           | [**@rvagg@twitter**](https://twitter.com/rvagg)               |
 | **Vincent Weevers**        | [**@vweevers**](https://github.com/vweevers)                     | [**@vweevers@twitter**](https://twitter.com/vweevers)         |
-| **Julian Gruber**          | [**@juliangruber**](https://github.com/juliangruber)             | [**@juliangruber@twitter**](https://twitter.com/juliangruber) |
+| **Rod Vagg**               | [**@rvagg**](https://github.com/rvagg)                           | [**@rvagg@twitter**](https://twitter.com/rvagg)               |
 | **jess**                   | [**@monkeywithacupcake**](https://github.com/monkeywithacupcake) |                                                               |
+| **Julian Gruber**          | [**@juliangruber**](https://github.com/juliangruber)             | [**@juliangruber@twitter**](https://twitter.com/juliangruber) |
 | **Tim Oxley**              | [**@timoxley**](https://github.com/timoxley)                     | [**@secoif@twitter**](https://twitter.com/secoif)             |
+| **Andrew Kelley**          | [**@andrewrk**](https://github.com/andrewrk)                     |                                                               |
+| **Joseph Dykstra**         | [**@ArtskydJ**](https://github.com/ArtskydJ)                     |                                                               |
 | **Matt Walters**           | [**@mateodelnorte**](https://github.com/mateodelnorte)           |                                                               |
 | **RasmusErik Voel Jensen** | [**@rasmuserik**](https://github.com/rasmuserik)                 |                                                               |
-| **Joseph Dykstra**         | [**@ArtskydJ**](https://github.com/ArtskydJ)                     |                                                               |
-| **Andrew Kelley**          | [**@andrewrk**](https://github.com/andrewrk)                     |                                                               |
```

Comment:
FYI, [`contributors-from-git@1.0.1`](https://github.com/vweevers/contributors-from-git) fixed the order of commits, and this floated into `remark-git-contributors` and `hallmark`, but the order is still a bit unstable. That will be fixed once https://github.com/remarkjs/remark-git-contributors/pull/31 is released.